database/schema.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE trains(
+  id SERIAL PRIMARY KEY,
+  capacity INTEGER,
+  current_riders INTEGER,
+  current_station VARCHAR(25)
+);
+
+CREATE TABLE passengers(
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(25),
+  ticket VARCHAR(25),
+  current_station VARCHAR(25),
+  current_train_id INTEGER
+);
```

Comment:
The separation of the schema makes sense after inspecting the package.json, but I think it could have all been in one file. The migrate schema and the schema files only differ by a table, and the drop statements could be added at the top of each create statement. In the end, it is more of a style choice.